### DISCUSSION TITLE
Time out error

### DISCUSSION THREAD

**Discussion by GuttoF** [AlertaDengue/PySUS#215] [Q&A] [answered]
Olá, tento usar o pysus desde terça e não estou conseguindo, sempre aparece esse erro aqui:
<img width="692" alt="Screenshot 2024-11-29 at 00 27 42" src="https://github.com/user-attachments/assets/e6d63215-217e-4345-991f-42f3be98f1f3">

Alguma dica de como resolver?

**Comment by felipeadeildo** [comment]
> Olá, tento usar o pysus desde terça e não estou conseguindo, sempre aparece esse erro aqui:
> <img width="692" alt="Screenshot 2024-11-29 at 00 27 42" src="https://github.com/user-attachments/assets/e6d63215-217e-4345-991f-42f3be98f1f3">
> 
> Alguma dica de como resolver?

Opa, bom dia, qual módulo especificamente você está tentando utilizar?

Qual a versão da biblioteca você está utilizando?

Como trata-se de timeout, talvez seja o mesmo que ocorreu aqui na discussão #211

**Comment by GuttoF** [comment]
Bom dia!

É o ftp.databases.sinan, acho que não é o mesmo problema, estou rodando localmente e continua a mesma coisa, tanto no macos quanto no linux com ip nacional.
<img width="382" alt="Screenshot 2024-11-29 at 09 02 12" src="https://github.com/user-attachments/assets/ad46390d-8cc5-460c-b15f-95e7fa2881c6">
<img width="465" alt="Screenshot 2024-11-29 at 09 01 57" src="https://github.com/user-attachments/assets/c72a1248-7cfa-4d08-bb40-a23dcfbf87c1">

**Reply by felipeadeildo** [reply, marked as answer]
Opa! Foi mal a demora.

Em resumo, como o @wrath-codes já respondeu, não trata-se de um problema com a biblioteca, o servidor FTP está offline realmente.

Vide imagem:

![image](https://github.com/user-attachments/assets/30650aa8-b3d0-42da-8531-4ceee3ca0b1a)

Não tem notícias sobre, nos resta esperar hahaa.

**Reply by GuttoF** [reply]
haha, vou ficar aguardando então! Muito obg pela ajuda @wrath-codes  e @felipeadeildo

**Comment by wrath-codes** [comment]
O servidor ftp nao esta no ar...
Nao acredito que o proplema seja com a biblioteca
